AdminApi.Tests.Integration/Features/Clients/GET.feature

```diff
@@ -0,0 +1,9 @@
+﻿@Integration
+Feature: GET Clients
+
+User requests an Client List
+
+Scenario: Requesting an Client List
+	When a GET request is sent to the Clients/ endpoint
```

Comment:
Can you do me a favour and change the route format in all tests so that there is a slash in front of the route instead of after it?
So in this example, instead of "Clients/" it should be "/Clients".